packages/plugins/RSS3/src/SNSAdaptor/components/FeedCard/NoteCard.tsx

```diff
@@ -124,8 +125,20 @@ const cardTypeMap = {
     [Type.Share]: CardType.NoteLink,
 } as const
 
-function resolveLenstubeWatchLink(publicationId: string) {
-    return `https://lenstube.xyz/watch/${publicationId}`
+const toHex = (num: number) => {
```

Comment:
Try this?

```ts
import { toHex } from 'web3-utils'
```

Reply:
Need to pad zero as well.

Reply:
```ts
import { padLeft, toHex } from 'web3-utils'
```

Reply:
`toHex` of `web3-util` returns `0xXX`, but we need to pad zero to `XX` to turn into such format `0xXX` or `0x0XXX`.

Besides, is it worth to import such two modules to do such simple thing?

Reply:
It's widely used.